src/components/xDragBoxLayer.ts

```diff
@@ -17,7 +17,7 @@ export module Components {
 
     public computeLayout(origin?: Point, availableWidth?: number, availableHeight?: number) {
       super.computeLayout(origin, availableWidth, availableHeight);
-      this.bounds(this.bounds()); // set correct bounds when width/height changes
+      this._setBoundsWithoutChangingMode(this.bounds()); // set correct bounds when width/height changes
```

Comment:
Note that I did not write the former code.  I can fix that code, but that is an outside concern.